processes-endpoint.md

```diff
@@ -141,134 +141,169 @@ In a future version, a solution supporting a live-feed can also be included.
-        "self": {
-          "href": "/api/core/bitstreams/e2343390-0b1b-4397-be3d-66fed243a436"
-        }
-    },
-      "_embedded": {
-        "format": {
-          "id": 1,
-          "shortDescription": "Text",
-          "description": "Plain Text",
-          "mimetype": "text/plain",
-          "supportLevel": 0,
-          "internal": false,
-          "extensions": null,
-          "type": "bitstreamformat",
+      {
+          "id": "e2343390-0b1b-4397-be3d-66fed243a436",
+          "uuid": "e2343390-0b1b-4397-be3d-66fed243a436",
+          "name": "map.txt",
+          "handle": null,
+          "metadata": {
+            "dc.title": [
+              {
+                "value": "map.txt",
+                "language": null,
+                "authority": null,
+                "confidence": -1,
+                "place": 0
+              }
+            ],
+            "dspace.process.filetype": [
+              {
+                "value": "mappingFile",
+                "language": null,
+                "authority": null,
+                "confidence": -1,
+                "place": 0
+              }
+            ]
+          },
+          "bundleName": null,
+          "sizeBytes": 479381,
+          "checkSum": {
+            "checkSumAlgorithm": "MD5",
+            "value": "64cdb4b4dd01b4a4cbb41e6d879fe996"
+          },
+          "sequenceId": null,
+          "type": "bitstream",
           "_links": {
+            "content": {
+              "href": "/api/core/bitstreams/e2343390-0b1b-4397-be3d-66fed243a436/content"
+            },
+            "format": {
+              "href": "/api/core/bitstreams/e2343390-0b1b-4397-be3d-66fed243a436/format"
+            },
             "self": {
-              "href": "/api/core/bitstreamformats/6"
+              "href": "/api/core/bitstreams/e2343390-0b1b-4397-be3d-66fed243a436"
+            }
+        },
+          "_embedded": {
+            "format": {
+              "id": 1,
+              "shortDescription": "Text",
+              "description": "Plain Text",
+              "mimetype": "text/plain",
+              "supportLevel": 0,
+              "internal": false,
+              "extensions": null,
+              "type": "bitstreamformat",
+              "_links": {
+                "self": {
+                  "href": "/api/core/bitstreamformats/6"
+                }
+              }
             }
           }
-        }
-      }
+      } 
+    ]
+    },
+    "page": {
+       "number": 0,
+       "size": 10,
+       "totalPages": 1,
+       "totalElements": 5
+    },
+    "_links": {
+       "self": {
+           "href": "/api/system/processes/10/files?page=0&size=10"
+       }
+    }
+}
+```
+
+## Get process file types
+
+**GET /api/system/processes/<:process-id>/filetypes**
+
+Files can have a different "type", the type can be anything it refers to what the file represents, is it a mapping file, an exported csv file, ...
+This endpoint will return all the different types  
```

Comment:
Tiny language cleanup. I'd recommend rewording this to say something like:

```
Individual processes can have files of different "types".  The types can be anything. It refers to what the file represents: e.g. mappingFile, exportCSV, etc.
This endpoint will return all the different file types of the specified process.  The `/api/system/processes/<:process-id>/files/<:type>` endpoint can then be used to find the file of that type.
```

Reply:
Cleared up the wording a bit.